__tests__/app.test.js

```diff
@@ -222,4 +222,103 @@ describe('app', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .expect(200)
+            .send({
+                inc_votes: 10
+            })
+            .then((response) => {
+                const result = response.body;
+                expect(typeof result).toBe('object');
+            })
+        })
+        test('status: 200 and responds with an object with a key of article', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .expect(200)
+            .send({
+                inc_votes: 10
+            })
+            .then((response) => {
+                const result = response.body;
+                expect(result).toHaveProperty('article');
+            })
+        })
+        test('status: 200 and responds with a nested article object with the correct keys', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .expect(200)
+            .send({
+                inc_votes: 10
+            })
+            .then((response) => {
+                const result = response.body.article;
+                expect(result).toHaveProperty("article_id");
+                expect(result).toHaveProperty("title");
+                expect(result).toHaveProperty("topic");
+                expect(result).toHaveProperty("author");
+                expect(result).toHaveProperty("body");
+                expect(result).toHaveProperty("created_at");
+                expect(result).toHaveProperty("votes");
+                expect(result).toHaveProperty("article_img_url");
+            })
+        })
+        test('status: 200 and responds with a nested article object with the updated votes value when newVote is a positive integer', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .expect(200)
+            .send({
+                inc_votes: 10
+            })
+            .then((response) => {
+                const result = response.body.article;
+                expect(result.votes).toBe(110);
+            })
+        })
+        test('status: 200 and responds with a nested article object with the updated votes value when newVote is a negative integer', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .expect(200)
+            .send({
+                inc_votes: -25
+            })
+            .then((response) => {
+                const result = response.body.article;
+                expect(result.votes).toBe(75);
+            })
+        })
+        test('status: 400 when the request body is empty', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .expect(400)
+            .send({})
+            .then((response) => {
+                const msg = response.body.msg;
+                expect(msg).toBe('Request body cannot be empty');
+            })
+        })
+        test('status: 400 when inc_votes is invalid', () => {
```

Comment:
tests to add. 
 [-] Status 400, invalid ID type, eg. /api/articles/handbag
 [-] Status 404, non existent ID, e.g. 0 or 999909090